attestation-service/src/policy_engine/mod.rs

```diff
@@ -75,7 +75,10 @@ type PolicyDigest = String;
 
 #[derive(Debug)]
 pub struct EvaluationResult {
-    pub rules_result: Value,
+    pub trust_claims: Value,
+    /// Extensions to be added to the attestation token.
+    /// The extension name, extension index, and the Value.
+    pub extensions: Vec<(String, i32, Value)>,
```

Comment:
Can we define a single struct for this? btw, could we provide an example rego to include the extension, in either document or a unit test?

Reply:
Ok, I changed to a struct and updated the docs. 

I prefer having `trust_claims` and `extensions` as two totally separate claims.